bundles/core/src/main/java/com/adobe/cq/forms/core/components/internal/form/FormConstants.java

```diff
@@ -57,6 +57,8 @@ private FormConstants() {
     /** The resource type for panel v1 */
     public static final String RT_FD_FORM_PANEL_V1 = RT_FD_FORM_PREFIX + "panel/v1/panel";
 
+    public static final String RT_FD_FORM_PANEL_CONTAINER_V1 = RT_FD_FORM_PREFIX + "panelcontainer/v1/panelcontainer";
```

Comment:
what is the difference between panel and panelcontainer

Reply:
all tab, accordion and panel container (responsive) are all implementations of panel